TypeScript.YAML-tmLanguage

```diff
@@ -23,14 +23,14 @@ repository:
 
   control-statement:
     name: keyword.control.ts
-    match: \b(break|catch|continue|debugger|declare|do|else|finally|for|if|return|switch|throw|try|while|with|super)\b
+    match: \b(break|catch|continue|debugger|declare|do|else|finally|for|if|return|switch|throw|try|while|with|super|case|default)\b
```

Comment:
I originally thought we wanted to keep it highlighted as keywords as Sublime does that for JS files.